### PR TITLE
Feature/unit test for add payloads alt 103

### PR DESCRIPTION
**Pull request by blackyblack**

### CODE REVIEW

File: core/src/test/java/org/veriblock/sdk/ServiceSecurityPayloadTests.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+import org.veriblock.sdk.models.VeriBlockPublication;
+import org.veriblock.sdk.models.VerificationException;
+import org.veriblock.sdk.services.SerializeDeserializeService;
+import org.veriblock.sdk.transactions.VeriBlockTransactionsAtv;
+import org.veriblock.sdk.util.ParseBlocks;
+import org.veriblock.sdk.util.Utils;
+
+public class ServiceSecurityPayloadTests {
+	
+	private static VeriBlockSecurity security;
```
Comment (eu321) [inline core/src/test/java/org/veriblock/sdk/ServiceSecurityPayloadTests.java]: random tabs instead of spaces in multiple locations

**Review by eu321 — APPROVED** [review]

File: core/src/test/java/org/veriblock/sdk/ServiceSecurityPayloadTests.java [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
...0201DDD3BBCAD76B681BC2A1E8453C80EBA154978C13B8C044B3173EBCC7CF27C871A3EF32596E74CF1718893EE6A65DE54D7E0405F5E10B3020194000019CED00020F344EB4782ACECF6EF1AB67453C80EBA154978C13B8C044B3173EBCC7CFADFAA159D73F0128BEC9A0BA09BCC67F5DE54D990405F5E10F18FA024000019CEE0002031A7EE5B9CDCF040DCEC2EF453C80EBA154978C13B8C044B3173EBCC7CFA9CB475021388EBB8F304F33852BBB285DE54DBC0405F5E11026120A4000019CEF000200C0161E988A0F30ACEBDE0B453C80EBA154978C13B8C044B3173EBCC7CF320F4D720587FC39A299D00476B078CD5DE54EB30405F5E124F064604000019CF00002B0E52A13C61B5D2BC95030AF453C80EBA154978C13B8C044B3173EBCC7CF5D83E7B7D30DA514FA580D91563E591B5DE54ECB0405F5E1537CE4A94000019CF100023C7C7D26528E6A9E75947316453C80EBA154978C13B8C044B3173EBCC7CFBCAA574FCCA4DDECF22928931E98E4B55DE54EE80405F5E1276AB8854000019CF200021F059032E4E3DB3C9542859F453C80EBA154978C13B8C044B3173EBCC7CF0FAD624290F7D0883AC85DD417CB89DA5DE54F380405F5E1388BF5004000019CF30002342586773CC168F894BB8630453C80EBA154978C13B8C044B3173EBCC7CF6C9A115D7355B6F38A014585A9633A9C5DE54F9E0405F5E13767E2804000019CF40002C8640A0753B18CB2AD86F746453C80EBA154978C13B8C044B3173EBCC7CF89AA14DBF6D21C2CD6CE8F8AD85DD45F5DE54FCC0405F5E13CD6E2744000019CF50002DBBD55AE2BB18CB5217BE6E5453C
```
Comment (Warchant) [inline core/src/test/java/org/veriblock/sdk/ServiceSecurityPayloadTests.java:107]: will test pass if you enable difficulty validation?

File: core/src/test/java/org/veriblock/sdk/ServiceSecurityPayloadTests.java (hunk 1 above)
Comment (blackyblack) [reply core/src/test/java/org/veriblock/sdk/ServiceSecurityPayloadTests.java]: fixed

File: core/src/test/java/org/veriblock/sdk/ServiceSecurityPayloadTests.java (hunk 2 above)
Comment (blackyblack) [reply core/src/test/java/org/veriblock/sdk/ServiceSecurityPayloadTests.java:107]: no

**Comment by blackyblack** [comment]
WIP

**Comment by blackyblack** [comment]
DONE

**Pull request merged**